### ISSUE TITLE
Update Ubuntu PPA for Eoan and Focal

### ISSUE THREAD

**Issue by joeSeggiola** [powertab/powertabeditor#287]
The newest release on the Ubuntu PPA is beta 10, and it's only available up until Xenial (18.04). I tried to force-install it on my Eoan (19.10), but it fails with the error below. Also Focal (20.04) came out recently.

```
I seguenti pacchetti hanno dipendenze non soddisfatte:
 powertabeditor : Dipende: libboost-filesystem1.58.0 ma non è installabile
                  Dipende: libboost-iostreams1.58.0 ma non è installabile
                  Dipende: libboost-program-options1.58.0 ma non è installabile
                  Dipende: libboost-regex1.58.0 ma non è installabile
                  Dipende: libboost-system1.58.0 ma non è installabile
                  Dipende: librtmidi2v5 ma non è installabile
                  Dipende: libboost-date-time1.58.0 ma non è installabile
```

**Comment by cameronwhite** [comment]
I'll look into this. I think I lost the VM where I did all the Ubuntu packaging work previously so I'll need to figure out how to get it set up again

**Comment by alastair5555** [comment]
I have the latest long term version of Ubuntu installed and I added powertabeditor to my sources but get an error because there is not a repository for my ubuntu version. Here is the error when trying to to an update:

$ sudo apt-get update -y

Hit:1 http://ca.archive.ubuntu.com/ubuntu bionic InRelease
Get:2 http://ca.archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]   
Ign:3 http://ppa.launchpad.net/powertab-developers/powertabeditor-stable/ubuntu bionic InRelease
Get:4 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]    
Get:5 http://ca.archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB] 
Err:6 http://ppa.launchpad.net/powertab-developers/powertabeditor-stable/ubuntu bionic Release
  404  Not Found [IP: 91.189.95.83 80]
Reading package lists... Done                                                  
E: The repository 'http://ppa.launchpad.net/powertab-developers/powertabeditor-stable/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

I hope you can build the latest version for 18.04 and higher if possible.

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

**Comment by cameronwhite** [comment]
There should now be a package available for `focal` (20.04). I'm working on bionic (18.04) but it's a bit more involved due to having older version of CMake and certain library dependencies

**Comment by cameronwhite** [comment]
A bionic package should also be published now
